tqp/topic_queue_poller.py

```diff
@@ -17,27 +17,43 @@ def _jsonify_dictionary(dictionary):
 # -----------------------------------------------------------------------------
 
 
-def create_queue(queue_name, *, tags, **kwargs):
-    sqs = boto3.resource("sqs")
+def _create_queue_raw(name, attributes, *, tags):
+    sqs_client = boto3.client("sqs")
+    sqs_resource = boto3.resource("sqs")
 
-    def _create_queue(name, attributes, extra_tags=None):
-        extra_tags = extra_tags or {}
+    attributes = _jsonify_dictionary(attributes)
+    tags = {"tqp": "true", **tags}
 
-        return sqs.create_queue(
-            QueueName=name,
-            Attributes=_jsonify_dictionary(attributes),
-            tags={"tqp": "true", "dlq": "false", **tags, **extra_tags},
+    def _create_queue():
+        return sqs_resource.create_queue(
+            QueueName=name, Attributes=attributes, tags=tags,
         )
 
-    dead_letter_queue = _create_queue(
+    try:
+        return _create_queue()
+    except sqs_client.exceptions.QueueNameExists:
```

Comment:
so this will only get hit if the existing tags are different?

Reply:
no, also if attributes are different. I don't want to parse the text of the error message though, so I'd rather adjust the tags and retry

Reply:
okay, i'm just saying – it won't get hit every time?

Reply:
nope